src/infrastructure/remote/team.ts

```diff
@@ -343,6 +343,22 @@ export function teamDataRemote(): TeamService {
     });
   };
 
+  const getTeamEditMember = async (teamID: number) => {
+    const response = await privateAPI
+      .get({ url: `/team/edit/member/${teamID}` })
+      .catch((error: AxiosError) => {
+        if (error.response?.status === STATUS_CODE.FORBIDDEN)
+          throw new ForbiddenError('팀 호스트에게만 팀 수정 권한이 있습니다.');
+      });
+    if (response.status === STATUS_CODE.OK) {
+      const { member: memberList } = response.data;
+      return memberList.map((member: any) => {
+        const { id, name, profileId: profileID, image, isConfirmed } = member;
+        return { id, name, profileID, image, isConfirmed };
```

Comment:
요거 그대로 리턴해도 될듯 합니다!
```suggestion
        return { ...member, profileID: member.profileId };
```
는 사실 상관 없을 것 같습니다!! (변덕쟁이)
마음에 드는 것으로 하셔요~

Reply:
이렇게 하면 필요 없는 profileId가 생겨서요!!